src/libs/actions/Report.js

```diff
@@ -1232,10 +1232,11 @@ function deleteReportComment(reportID, reportAction) {
  *
  * @param {Number} reportID
  * @param {Number} [sequenceNumber] This can be used to set the last read actionID to a specific
+ * @param {Boolean} manuallyMarked If the user manually marked this as unread, we need to tell the API
  *  spot (eg. mark-as-unread). Otherwise, when this param is omitted, the highest sequence number becomes the one that
  *  is last read (meaning that the entire report history has been read)
  */
-function updateLastReadActionID(reportID, sequenceNumber) {
+function updateLastReadActionID(reportID, sequenceNumber, manuallyMarked = false) {
```

Comment:
**opinion (non blocker):** creating a more specific method like `manuallyUpdateLastReadActionID()` can be clearer sometimes and save you from having to use a boolean args that can be ambiguous. that said, we use them a lot :D